src/Gzero/Repository/BaseRepository.php

```diff
@@ -63,7 +63,7 @@ public function getById($id)
      */
     public function getDeletedById($id)
     {
-        return $this->newORMQuery()->onlyTrashed()->find($id);
+        return $this->newORMQuery()->withTrashed()->find($id);
```

Comment:
I don't understand this change, why getDeletedById is returning all records no mater if it's deleted or not? It should return only deleted records